### ISSUE TITLE
driver.header_overrides doesn't set all headers

### ISSUE THREAD

**Issue by jedie** [wkeeling/selenium-wire#8]
Test script:

```
import logging
from pprint import pprint
from seleniumwire import webdriver 

logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.DEBUG)

chrome_options = webdriver.ChromeOptions()
chrome_options.add_argument("--no-sandbox")
chrome_options.add_argument("--headless")
chrome_options.add_argument("--disable-gpu")

driver = webdriver.Chrome(
    chrome_options=chrome_options,
    executable_path="/usr/lib/chromium-browser/chromedriver",
)

driver._client.set_header_overrides(
    headers={
        "User-Agent": "Mozilla/5.0 (test)",
        "Accept": "text/html, foo/bar",
        "Accept-Language": "en,de;q=0.7,fr;q=0.3",
        'New-Header1': 'Some Value',
    })

driver.get("https://www.google.com")

for request in driver.requests:
    if request.response:
        print(request.path, request.response.status_code)
        pprint(dict(request.headers))
```

output from first request:
```
https://www.google.com/ 200
{'Accept': 'text/html, foo/bar',
 'Accept-Encoding': 'gzip, deflate',
 'Connection': 'keep-alive',
 'Host': 'www.google.com',
 'Upgrade-Insecure-Requests': '1',
 'User-Agent': 'Mozilla/5.0 (test)'}
```
So: "Accept" and "User-Agent" are set, but not "Accept-Language" and "New-Header1" ...

Maybe related to chrome usage ?!?

**Comment by wkeeling** [comment]
@jedie - thanks, I'll check this one out and see what's going on.

**Comment by wkeeling** [comment]
@jedie This was a bug affecting all browsers, which is now fixed by ebdcc66.

I have released version 0.8.0 of selenium-wire to PyPi which contains the fix.

Many thanks for reporting!
